src/Reflection/ReflectionProperty.php

```diff
@@ -156,13 +156,16 @@ public static function createFromInstance(object $instance, string $propertyName
     /** @internal */
     public function withImplementingClass(ReflectionClass $implementingClass): self
     {
-        $clone                    = clone $this;
+        $clone = clone $this;
+        /** @phpstan-ignore-next-line property.readOnlyByPhpDocAssignNotInConstructor */
```

Comment:
After reading through a bunch of these, I'd say that PHPStan does not currently understand our `with` design, and we should suppress this family of errors at configuration level

Reply:
Done.